app/views/lookup_keys/_values.html.erb

```diff
@@ -17,10 +17,11 @@
   </thead>
 
   <tbody>
-  <%= f.fields_for :lookup_values do |lookup_values| %>
-    <%= render 'lookup_keys/value', :f => lookup_values, :is_param => is_param, :is_template => false %>
+  <%= f.fields_for :lookup_values do |lookup_value| %>
+    <%= render 'lookup_keys/value', :f => lookup_value, :is_param => is_param, :is_template => false, :hidden_value => f.object.hidden_value? %>
```

Comment:
This is being checked already in the js if `:hidden_value` is nil, why pass it explicitly?